tests/core/test_evaluation.py

```diff
@@ -495,3 +495,78 @@ def test_log_evaluation_table(caplog, skip_field, skip_value):
         assert f"Classification report: \n{kwargs['report']}" in caplog.text
     else:
         assert "Classification report:" not in caplog.text
+
```

Comment:
I added three tests because my change also affects if a comment is printed in the `intent` line. Before these changes, we always printed a comment in the `intent` line regardless if the prediction error was in the `intent` or the `entities`. Now we print only where the wrong prediction so : 
- If intent is correct and there is a wrongly predicted entity -> we print a comment only on entity line
- if intent is wrongly predicted and entities correct -> we print only on intent line
- if both are wrongly predicted --> we print on intent and entity line

That is why I added the 3 separate tests that corresponds to each case.